assets/js/components/GoogleChart.js

```diff
@@ -169,7 +171,7 @@ export default function GoogleChart( props ) {
 
 			clearInterval( interval );
 		};
-	}, [] );
+	} );
```

Comment:
I've assumed that any `useEffect` with an empty dependency array is meant to only run on mount (as opposed to being an error!)